rollup.config.js

```diff
@@ -0,0 +1,23 @@
+import ts from 'rollup-plugin-ts';
+import pkg from './package.json' assert { type: 'json' };
```

Comment:
I'm not sure about this anymore these days because it will be replaced by:
```
import pkg from './package.json' with { type: 'json' };
```

See: https://github.com/tc39/proposal-import-attributes#history

Turns out, it is now in stage 2 again, for me personally stage 3 is the minimum something needs to be at before I'm willing to use it.. and even though import assertions were in stage 3 and implemented in browsers and node, it was demoted back to stage 2, renamed, and now only has a consensus since last month to go back to stage 3.

Perhaps better to just go with the good old `JSON.parse(fs.readFileSync(filePath, 'utf-8'))` for now? Also saves us from needing the `resolveJsonModule` compiler option in tsconfig I think? Anyway, not a hill for me to die on, so take it or leave it :)

Reply:
Yeah I didn't want to have to change the strict mode to support the `with` directive and I liked the type safety and intellisense when I used `resolveJsonModule`